stable/redis/templates/redis-master-svc.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Service
+metadata:
+  {{- if .Values.cluster.enabled }}
```

Comment:
I don't think this is needed, lets just name it `{{ template "redis.fullname" . }}-master` at all times. Better to avoid this complication